crawl-ref/source/dat/des/branches/temple.des

```diff
@@ -2794,6 +2794,64 @@ llllllClllllllllllllllllllEllllllllDlllllllllllllll
 llllllClllllllllllllllllllEllllllllDlllllllllllllll
 ENDMAP
 
+NAME:    mainiacjoe_temple_archimedes_tessellation
+PLACE:   Temple
+ORIENT:  encompass
+WEIGHT:  10
+SUBST:   c : cvb
+SHUFFLE: lw
+MAP
```

Comment:
A question of aesthetics: the big areas of deep water can have shallow water added, so instead of big blocks of red and blue, you get big blocks of red with blocks of mixed blues, which looks off. If this isn't the intent, I would add TAGS: no_pool_fixup to the vault definition.